sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -458,6 +464,69 @@ def _is_live(self, action):
         except TypeError:
             pass
 
+    def _settle_message(
+            self,
+            settle_operation,
+            dead_letter_details=None
+    ):
+        try:
+            if not self._is_deferred_message:
+                try:
+                    self._get_settle_via_receiver_link(settle_operation, dead_letter_details)()
```

Comment:
If we're building the function JUST to immediately call it, it would seem to make more sense to just "_settle_via_receiver_link"/"_settle_via_mgmt_link"?

Reply:
updated!